spring-cloud-gcp-pubsub/src/test/java/org/springframework/cloud/gcp/pubsub/integration/inbound/PubSubInboundChannelAdapterTests.java

```diff
@@ -65,31 +64,30 @@
 
 	private String value;
 
-	private BasicAcknowledgeablePubsubMessage message;
+	private ConvertedBasicAcknowledgeablePubsubMessage message;
```

Comment:
Is there some consumer of this property?
I mean do we really need to have it on the class level, but not locally only in the `setUp()` ?